lti_consumer/lti_consumer.py

```diff
@@ -740,6 +799,85 @@ def student_view(self, context):
         fragment.initialize_js('LtiConsumerXBlock')
         return fragment
 
+    has_author_view = True
+
+    def author_view(self, context):
+        """
+        XBlock studio view of this component.
+
+        Makes a request to `lti_launch_handler` either
+        in an iframe or in a new window depending on the
+        configuration of the instance of this XBlock
+
+        Arguments:
+            context (dict): XBlock context
+
+        Returns:
+            xblock.fragment.Fragment: XBlock HTML fragment
+        """
+        if self.lti_version != "lti_1p3":
+            return self.student_view(context)
+
+        fragment = Fragment()
+        loader = ResourceLoader(__name__)
+        context = {
+            "client": self.lti_1p3_client_id,
+            "deployment_id": "1",
```

Comment:
Does this deployment_id need to be replaced with a "real" value instead of `"1"`?

Reply:
@davidjoy The current implementation uses `Single tenant` use case described in the core specification here: http://www.imsglobal.org/spec/lti/v1p3/#single-tenant-tool-registered-and-deployed-once.

![deployments_onetoone_bw](https://user-images.githubusercontent.com/27893385/79789214-3cc5a780-8320-11ea-9b0a-dc579dd7a851.png)

So the deployment ID is one for all deployments, but this would be different if the XBlocks shared the same key and client ID, which shouldn't happen.